app/Http/Controllers/Api/UnsubscribeController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Services\Manager;
+use Gladiator\Models\Competition;
+use Gladiator\Http\Requests\UnsubscribeRequest;
+
+class UnsubscribeController extends ApiController
+{
+    /**
+     * @var Gladiator\Services\Manager;
+     */
+    protected $manager;
+
+    public function unsubscribe(UnsubscribeRequest $request)
+    {
+        $credentials = $request->all();
+
+        $competition = Competition::find($credentials['competition_id']);
```

Comment:
I don't think you need to load the whole competition here (or below) as you won't be doing anything with it.